Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.3
 
 RUN apk --no-cache add \
-    ca-certificates
+    ca-certificates iptables
```

Comment:
Please go to the line after `ca-certificates` and use `\` to separate each line.